Python/Product/PyDebugAttach/PyDebugAttach.cpp

```diff
@@ -324,16 +324,17 @@ __success(return) BOOL EnumProcessModulesHelper(
                     if (curTable->DllBase == nullptr) {
                         break;
                     }
-                    lphModule[(*lpcbNeeded) / sizeof(SIZE_T)] = (HMODULE)curTable->DllBase;
+                    *lphCurrentModule = (HMODULE)curTable->DllBase;
+                    lphCurrentModule++;
                 }
 
                 (*lpcbNeeded) += sizeof(SIZE_T);
                 cur = (LDR_DATA_TABLE_ENTRY*)((LIST_ENTRY*)cur)->Flink;
             } while (cur != start && cur != 0);
 
-            return *lpcbNeeded <= cb;
+            return (lphCurrentModule == lphModule) ? FALSE : *lpcbNeeded <= cb;
```

Comment:
![image](https://user-images.githubusercontent.com/3315992/53908854-25850700-4005-11e9-94c3-cfb13a49630a.png)

Reply:
I'm confused about how this one fixes that. Isn't the real problem here that `lphModule` isn't properly annotated as an array with `cb` as size, and so static analysis expects there to be exactly one element written via it, no more and no less?

But also, this is pre-Win7 workaround. @huguesv, do we even support that still? If not, let's just delete this whole function, and the caller can then call `EnumProcessModules` directly.

Reply:
It appears that Win7 SP1 and Server 2012 R2 are the oldest OSes that we support:
https://docs.microsoft.com/en-us/visualstudio/releases/2019/system-requirements

If that's pre-Win7, then it could go away?

Reply:
Removed and directly calling EnumProcessModules.